openslides_backend/action/actions/user/forget_password.py

```diff
@@ -42,6 +34,16 @@ class UserForgetPassword(EmailMixin, UpdateAction):
     skip_archived_meeting_check = True
 
     def get_updated_instances(self, action_data: ActionData) -> ActionData:
+        _ = self.get_translate_function()
+        self.PW_FORGET_EMAIL_TEMPLATE = _(
+            """You are receiving this email because you have requested a new password for your OpenSlides-account.
+
+Please open the following link and choose a new password:
+{url}/login/forget-password-confirm?user_id={user_id}&token={token}
+
+For completeness your username: {username}"""
+        )
+        self.PW_FORGET_EMAIL_SUBJECT = _("Reset your OpenSlides password")
```

Comment:
I think you can also use `gettext` as just a marker function and do the actual translating later. Can you try this out, so you don't need to have hardcoded strings inside the class?

Reply:
The way I use gettext, it doesn't work with translation later.